Index: src/jogodavelha/Jogo.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package jogodavelha;\r\n\r\npublic class Jogo {\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/jogodavelha/Jogo.java b/src/jogodavelha/Jogo.java
--- a/src/jogodavelha/Jogo.java	
+++ b/src/jogodavelha/Jogo.java	
@@ -1,5 +1,5 @@
 package jogodavelha;
 
 public class Jogo {
-
+    
 }
